frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -583,6 +587,35 @@ pub mod pallet {
 		/// this value, based on [`WeightInfo::submit_unsigned`].
 		type MinerMaxWeight: Get<Weight>;
 
+		/// Maximum number of signed submissions that can be queued.
+		#[pallet::constant]
+		type SignedMaxSubmissions: Get<u32>;
```

Comment:
I think it would be great to start mentioning how some configurable value can be changed,
here we should say that if there is an ongoing election, any decrease must update the storage as well.

Reply:
7b673d166